.github/workflows/minor_version_release.yml

```diff
@@ -36,11 +39,10 @@ jobs:
           NEW_VERSION=$(awk -F. '/[0-9]+\./{$2++;print}' OFS=. < VERSION.txt)
           echo "$NEW_VERSION" > VERSION.txt
           cat VERSION.txt
-          git checkout -b bump-version
           git add .
-          git commit -m "Update unstable version"
-          git push -u origin bump-version
-          gh pr create -B main -H bump-version --title 'Bump unstable version' --body 'Part of the release process' --label 'ignore-for-release-notes'
+          git commit -m "Update unstable version to $NEW_VERSION"
+          git tag v"$NEW_VERSION" -m"v$NEW_VERSION"
+          git push -u origin main --tags
```

Comment:
Will this actually work? `push --tags` doesn't pushes branches, but I've never tried with `-u` though. 🤔

Reply:
What about `--atomic`?

Reply:
That works. 👍